src/views/collection-view/helpers/get-slug.ts

```diff
@@ -15,30 +15,33 @@ export function getTutorialSlug(
 	tutorialDbSlug: string,
 	collectionDbSlug: string
 ): string {
-	const [product, collectionFilename] = collectionDbSlug.split('/')
+	const [rawProductSlug, collectionFilename] = collectionDbSlug.split('/')
```

Comment:
Refactored a bit here as `get-slug` needs to account for the `hcp` ←→ `cloud` difference between Tutorials and Dev Dot.